packages/moonstone/ContextualPopupDecorator/ContextualPopupDecorator.js

```diff
@@ -282,7 +282,7 @@ const Decorator = hoc(defaultConfig, (config, Wrapped) => {
 				this.positionContextualPopup();
 			}
 
-			if (this.props.open && !prevProps.open) {
+			if (this.props.open && this.state.activator && !prevState.activator) {
```

Comment:
This condition will also avoid entering this block when nothing is focused when opening the contextual popup (as is often the case in sampler) since `activator` could be set to `null` even though the floating layer is in fact open.

Reply:
In Sampler, activator sometimes became 'undefined'. (I can't find 'null') Even if it is 'undefined', the condition of the above if statement does not work as expected.

So I modified the code by adding a new state to your opinion. In this case, ContextualPopup modified the state value to be true after the node was drawn in the FloatingLayer.